src/site/content/en/blog/speculative-prerendering/index.md

```diff
@@ -429,6 +429,20 @@ Timer.html without speculation rules
   </figure>
 </div>
 
+## Prerendering from the address bar
+
+Starting with Chrome 101, prerendering will gradually be applied to the most likely navigation when a user triggers autocomplete suggestions by interacting with Chrome’s address bar. We are starting with Chrome for Android but intend to bring this feature to the other platforms we support in follow-up milestones.
+
+Before this launch, Chrome used [No-State Prefetch](#prerender-implementation-using-no-state-prefetch) to prefetch critical resources for the most likely navigations from the address bar. Prerendering takes things a bit further by rendering the page, which includes constructing the DOM tree and executing scripts.
+
+While this is a browser feature, it can have observable effects from the point of view of the prerendered websites. For instance, APIs that may cause unexpected surprises are delayed until the user navigates to the prerendered web page (e.g., text to speech from an invisible website). 
```

Comment:
```suggestion
While this is a browser feature, it can have observable effects from the point of view of the prerendered websites. For instance, APIs that may cause unexpected surprises are delayed until the user navigates to the prerendered web page (for example, text to speech from an invisible website). 
```